src/components/Panel/Panel.stories.js

```diff
@@ -3,145 +3,236 @@
+                        );
+                      })}
+                  </ComboButton>
+                )}
+              >
+                <PanelContent>
+                  <Fragment>
+                    {content}
+                    <p>
+                      This example uses the
+                      <CodeSnippet type="inline" light>
+                        renderFooter
+                      </CodeSnippet>
+                      prop to render a
+                      <CodeSnippet type="inline" light>
+                        ComboButton
+                      </CodeSnippet>
+                      in a custom footer.
+                    </p>
+                  </Fragment>
+                </PanelContent>
+              </Panel>
+              <Panel
+                key="p3"
+                isOpen={this.state.thirdOpen}
+                {...props}
                 closeButton={{
-                  onClick: toggleActive,
+                  onClick: this.closethird,
                   label: closeButtonLabel,
                 }}
               >
-                <PanelContent>{panelContent}</PanelContent>
-              </PanelContainer>
-            </PanelController>
-          </Fragment>
-        )}
-      />
-      <Panel
-        {...panelProps}
-        render={({ active, toggleActive }) => (
-          <Fragment>
-            <Button onClick={toggleActive}>{footerLabel}</Button>
-            <PanelController active={active}>
-              <PanelContainer
-                {...panelContainerProps}
-                closeButton={{
-                  onClick: toggleActive,
-                  label: text('closeButton.label', closeButtonLabel),
-                }}
-                renderFooter={() => <Button size="large">{footerLabel}</Button>}
-              />
-            </PanelController>
-          </Fragment>
-        )}
-      />
-    </Fragment>
-  ),
-
-  {
-    info:
-      "View the 'Story' addon panel for guidance on using the `renderFooter` prop",
-  }
-);
+                <PanelContent>
+                  <Fragment>
+                    {content}
+                    <p>
+                      This example intentially does not include the
+                      <CodeSnippet type="inline" light>
+                        renderFooter
+                      </CodeSnippet>
+                      or
+                      <CodeSnippet type="inline" light>
+                        primaryButton
+                      </CodeSnippet>
+                      props and therefore does not have a footer.
+                    </p>
+                  </Fragment>
+                </PanelContent>
+              </Panel>
+            </Fragment>
+          );
+        }
+      }
+      return <PanelStory />;
+    },
+    {
+      info: {
+        text: `
+        The \`Panel\` is an experimental component that is subject to change while it is being tested. If you need a stable component, please use the \`Panel\` instead.
+      `,
```

Comment:
Could you update this info text since the component is stable / not experimental in this context?

Reply:
Good catch!

Reply:
Good call!